cfme/tests/cloud/test_cloud_timelines.py

```diff
@@ -340,7 +348,8 @@ def test_cloud_timeline_diagnostic(new_instance, mark_vm_as_appliance, soft_asse
     inst_event.catch_in_timelines(soft_assert, targets)
 
 
-@pytest.mark.meta(blockers=[BZ(1537520, forced_streams=['5.8'])])
+@pytest.mark.meta(blockers=[BZ(1537520, forced_streams=['5.8']),
```

Comment:
same as above; master branch not supporting `5.8`. There is separate branch for `5.8` testing.